README.adoc

```diff
@@ -376,11 +376,11 @@ Whichever one you pick - apply it consistently.
 
 [source,ruby]
 ----
+# bad - no space after { and before }
+{one: 1, two: 2}
+
 # good - space after { and before }
 { one: 1, two: 2 }
-
-# good - no space after { and before }
-{one: 1, two: 2}
```

Comment:
Both are good cases in this style guide, so this is not typo.

Reply:
Got it. I've reverted this changes.